lib/assessment.rb

```diff
@@ -13,28 +13,36 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(&:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    if numbers.each.odd?
+      true
+    else
+      false
+    end
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    if numbers.each.even? == false
+      false
+    else
+      true
+    end
   end
 
   # Question 5
   # Write ruby code that returns an array with all of the
   # entries from the words enumerable capitalized.
   def transform(words)
-    # Your code goes here
+    return words.map { |a| a.upcase }
```

Comment:
Also, the `return` is not necessary here; by default, Ruby will always return the last evaluated expression in your method.